cmd/juju/application/deploy.go

```diff
@@ -364,16 +364,11 @@ If a revision is specified, a channel must also be specified for Charmhub charms
 and bundles.  The charm will be deployed with revision.  The channel will be used
 when refreshing the application in the future.
 
-A local charm may be deployed by giving the path to its directory:
-
-  juju deploy /path/to/charm
-  juju deploy /path/to/charm --base ubuntu@22.04
-
 You will need to be explicit if there is an ambiguity between a local and a
 remote charm:
 
-  juju deploy ./pig
-  juju deploy ch:pig
+  juju deploy ./postgresql
```

Comment:
`juju deploy ./postgreql.charm` helps to emphasize charm craft packed charm?

Reply:
Agreed.